src/AutoRest.CSharp/MgmtTest/TestCommon/Tool.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Collections.Generic;
+using System.Text;
+using Newtonsoft.Json;
```

Comment:
Where do we need this namespace? And are there possibilities to remove this dependency?

Reply:
removed